styles.css

```diff
@@ -3,15 +3,26 @@
 }
 
 body:not(footer) {
-  font-family: 'Montserrat', sans-serif;
+  font-family: 'Open Sans', sans-serif;
   font-size: 112.5%;
   line-height: 1.5;
-  background: linear-gradient(90deg, rgba(0, 138, 138, 7%),rgba(0, 138, 138, 1%), rgba(0, 138, 138, 7%));
-  color: rgba(0, 0, 0, 80%);
+  background-image: url(images/background-mountains.jpg);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  color: #081c24;
   margin: 8rem auto 2rem;
   padding: 2rem;
 }
 
+::-moz-selection {
+  color: lightblue; background: rebeccapurple;
+}
+
+::selection {
+  color: lightblue; background: rebeccapurple;
+}
+
```

Comment:
Hey Daniel, may I ask for what do you use it? I do not really get it even by checking google (just interested) :-)

Reply:
`::-moz-selection {
  color: lightblue; background: rebeccapurple;
}

::selection {
  color: lightblue; background: rebeccapurple;
}`

this if found to change the color for selected text and the background color